package-lock.json

```diff
@@ -3319,7 +3319,6 @@
       "version": "3.1.0",
       "resolved": "https://registry.npmjs.org/@ministryofjustice/frontend/-/frontend-3.1.0.tgz",
       "integrity": "sha512-Dp2xqgXNM2om56EQdjNXaEKju3k330sTRoXNMEQEHI2qYTSwyz67VW3PkUHrWH1BXE7E2MyU/Q3OVA6ZhcZBsA==",
-      "dev": true,
```

Comment:
Not sure why the FE module moved out of the dev dependencies to a main dependency.